src/query/graphite/native/aggregation_functions.go

```diff
@@ -135,6 +135,29 @@ func divideSeries(ctx *common.Context, dividendSeriesList, divisorSeriesList sin
 	return r, nil
 }
 
+// aggregate takes a list of series and returns a new series containing the
+// value aggregated across the series at each datapoint using the specified function.
+func aggregate(ctx *common.Context, series singlePathSpec, fname string) (ts.SeriesList, error) {
+	switch fname {
```

Comment:
Believe you're missing support for a few here? Namely:
- avg_zero
- median
- stddev
- range (or rangeOf)
- last (or current)

As per:
https://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.aggregate

Reply:
Ah, I was just checking the implemented methods in the aggregation_function.go, forgot to check the whole list in buildin_function.go. (stddev and range seems are implemented already) Will update the support for methods that are already implemented. 

For other aggregate functions that are not implemented, I will update the PR to document them for now.  and do some query internally to see if any of them are actually being used, if used, I will implement them.